root/etc/e-smith/templates/dnsmasq-instance/97Zones

```diff
@@ -12,7 +12,7 @@
 	foreach my $elem (@list_zone) { 	
 		$OUT .= "address=/$elem->{zone}/$elem->{ip_array}[0]\n";
```

Comment:
I think there's an issue here, what happend if i have only 1 ip address ?
This still return `address=xxx.xxx.xxx.xxx` instead of `server=xxx.xxx.xxx.xxx`